gendocs/env_constants.md

```diff
@@ -32,14 +32,17 @@ $ export QUILT_TRANSFER_MAX_CONCURRENCY=20
 ```
 
 ### `XDG_*`
-Quilt uses platformdirs for Python to determine where to write data. You can therefore
-override the following path constants with environment variables using the XDG
-standard (see [platformdirs docs](https://pypi.org/project/platformdirs/)).
+Quilt uses platformdirs for Python to determine where to write data. You can
+override the following environment variables from the XDG
```

Comment:
Same as the previous comment:

It doesn't make sense to say "override the following path constants with...." and then not list the path constants until the very end of the section (i.e. after an un-related example section). Move the "Constants" section immediately below this initial sentence, and then have the lambda example code in an "Example" section beneath the "Constants" section.